maas/plugins/horizon_check.py

```diff
@@ -111,5 +111,8 @@ def main(args):
         parser.add_argument('ip',
                             type=ipaddr.IPv4Address,
                             help='horizon dashboard IP address')
+        parser.add_argument('site_name_regexp',
+                            type=str,
+                            help='Horizon Site Name')
```

Comment:
Is it maybe worth having a default value of 'openstack dashboard' for this arg and making it optional? This way anyone already using this plugin won't have to make any changes to continue using it in the same way they already are (e.g. rpc-r)